src/Microsoft.DotNet.Wpf/src/PresentationFramework/Microsoft/Win32/FileDialog.cs

```diff
@@ -309,7 +163,7 @@ public string[] SafeFileNames
 
         //   If multiple files are selected, we only return the first filename.
         /// <summary>
-        ///  Gets or sets a string containing the full path of the file selected in 
+        ///  Gets or sets a string containing the full path of the file or folder selected in 
```

Comment:
Comment should be reverted here.

Reply:
Fixed in https://github.com/dotnet/wpf/pull/7916/commits/543c8098a03629f522d8d86a9666b673a0e0a045